app/src/main/java/com/codesoom/assignment/DemoHttpHandler.java

```diff
@@ -0,0 +1,133 @@
+            if (path.equals("/tasks/")) {
+                RequestSend(NOT_FOUND, exchange, "Please enter your ID");
+            } else {
+                for (Task task1 : tasks) {
+                    content = compareConvert(path, content, task1);
+                }
+                RequestSend(OK, exchange, content);
+            }
+        }
+
+        //일 생성
+        if (requestMethod.equals("POST") && path.equals("/tasks")) {
+            createNewTask(body);
+            RequestSend(CREATE, exchange, "Create a new task");
+        }
+
+        //수정
+        if (requestMethod.equals("PATCH") || requestMethod.equals("PUT") && path.startsWith("/tasks/")) {
+            updateTask(path, body);
+            RequestSend(OK, exchange, "Update task");
+        }
+
+        //삭제
+        if (requestMethod.equals("DELETE") && path.startsWith("/tasks/")) {
+            tasks.remove(getRequestId(path).intValue() - 1);
+            RequestSend(OK, exchange, "Delete Task");
+        }
+
+    }
+
+    private static void RequestSend(int statusCode, HttpExchange exchange, String content) throws IOException {
+        exchange.sendResponseHeaders(statusCode, content.getBytes().length);
+        OutputStream outputStream = exchange.getResponseBody();
+        outputStream.write(content.getBytes());
+        outputStream.flush();
+        outputStream.close();
+    }
+
+
+    private String compareConvert(String path, String content, Task task1) throws IOException {
+        if (task1.getId().equals(getRequestId(path))) {
+            content = tasksToJSON(task1);
+        }
+        return content;
+    }
+
+    private void updateTask(String path, String body) throws JsonProcessingException {
+        Task task = toTask(body);
+        task.setId(getRequestId(path));
+        tasks.set(getRequestId(path).intValue() - 1, task);
+    }
+
+    private void createNewTask(String body) throws JsonProcessingException {
+        Task task = toTask(body);
+        task.setId(autoIncrementId());
+        tasks.add(task);
+    }
+
+    private Long autoIncrementId() {
+        return defaultId += 1L;
+    }
+
+
+    private String getBody(HttpExchange exchange) {
+        return new BufferedReader(new InputStreamReader(exchange.getRequestBody()))
+                .lines()
+                .collect(Collectors.joining("\n"));
+    }
+
+    private Long getRequestId(String path) {
+        return Long.parseLong(path.split("/")[2]);
+    }
+
+    private Task toTask(String content) throws JsonProcessingException {
+        return objectMapper.readValue(content, Task.class);
+    }
+
+    private String tasksToJSON(Object object) throws IOException {
+        OutputStream outputStream = new ByteArrayOutputStream();
+        if (object.equals(tasks)) {
+            objectMapper.writeValue(outputStream, this.tasks);
+        } else {
+            objectMapper.writeValue(outputStream, object);
+        }
+        return outputStream.toString();
+    }
```

Comment:
```suggestion
    private String tasksToJSON(Object object) throws IOException {
        OutputStream outputStream = new ByteArrayOutputStream();
        objectMapper.writeValue(outputStream, object);
        return outputStream.toString();
    }
```

objectMapper.writeValue는 Object타입을 받고 있으니, 위와 같이 쓰고 tasks를 Json으로 바꾸고 싶다면 `tasksToJSON(tasks)`로 넘기면 여기 if문이 없어질 것 같은데 괜찮을까요